topics/nuget/index.md

```diff
@@ -1,13 +1,14 @@
 ---
 aliases: nuget-package
-created_by: Microsoft, .Net Foundation
+created_by: Microsoft, .NET Foundation
 display_name: NuGet
 github_url: https://github.com/NuGet
 logo: nuget.png
-released: October 2010
-short_description: NuGet is an open source package manager designed for the Microsoft development platform.
+related: dotnet
+released: October 5, 2010
+short_description: NuGet is the package manager for .NET.
 topic: nuget
-url: https://www.nuget.org/
+url: https://www.nuget.org
 wikipedia_url: https://en.wikipedia.org/wiki/NuGet
 ---
-NuGet is a free and open source package manager designed for the Microsoft development platform. It is formerly known as NuPack.
+NuGet is the package manager for .NET. The NuGet client tools provide the ability to produce and consume packages. The NuGet Gallery is the central package repository used by all package authors and consumers.
```

Comment:
NuGet is used for more than just .NET. Parts. Including "designed for the Microsoft development platform" would be more accurate.

Reply:
I have no idea. The Nuget [website](https://www.nuget.org/policies/About) and [docs](https://docs.microsoft.com/nuget/what-is-nuget) say Nuget is the Microsoft-supported mechanism for sharing code for .NET.

Reply:
@leecow I've made the changes. But I still want to know which other platforms it supports.